config/locales/diaspora/en.yml

```diff
@@ -732,10 +732,10 @@ en:
 
   notifier:
     a_post_you_shared: "a post."
-    a_private_message: "There's a new private message in diaspora* for you to check out."
-    a_limited_post_comment: "There's a new comment on a limited post in diaspora* for you to check out."
-    email_sent_by_diaspora: "This email was sent by %{pod_name}. If you'd like to stop getting emails like this,"
-    click_here: "click here"
+    a_private_message: "There’s a new private message in diaspora* for you to check out."
+    a_limited_post_comment: "There’s a new comment on a limited post in diaspora* for you to check out."
+    email_sent_by_diaspora: "This email was sent by %{pod_name}. If you’d like to stop getting emails like this,"
```

Comment:
Okay, weird issue.

The `’` is apparently triggering some additional escaping in the email output, which makes a spec fail. I guess the encoding is right and the spec should probably be updated, but since I only guess that let's work around it for now and use the HTML-Entity for this case (`&rsquo;`).